.travis.yml

```diff
@@ -57,11 +56,9 @@ matrix:
     stage: lint
   - python: pypy2.7-7.1.1
     env: TOXENV=pypy
-    dist: xenial
```

Comment:
I guess these are redundant due to the distribution global configuration setting: https://github.com/celery/celery/pull/5672/files#diff-354f30a63fb0907d4ad57269548329e3R2 ?

Reply:
Yes, you are right.